src/Dialogs/QuickGuide/ConfigurationWidget.cpp

```
@@ -0,0 +1,403 @@
+    canvas->Select(fontMono);
+    canvas->DrawText({x, y + icon_offset},
+                     (c5 == nullptr || StringIsEmpty(c5)) ? _T("[ ]") : _T("[X]"));
+  }
+  const TCHAR *t5 = _("Set your name.");
+  y += int(DrawTextBlock(fontDefault, x_text, t5)) + margin / 2;
+  const TCHAR *l5 = _("Config → System → Setup → Logger");
+  y += int(DrawLinkLine(LinkAction::SETUP_LOGGER_1, l5)) + margin;
+
+  // Pilot weight
+  if (canvas != nullptr) {
+    const char *c6 = Profile::Get(ProfileKeys::CrewWeightTemplate);
```

Cppcheck (2.19.0)

[error] Dereference of an invalid iterator

(derefInvalidIterator)